indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -730,13 +735,14 @@ private void notifyStatus(final TaskEntry entry, final TaskStatus taskStatus, St
       taskRunner.shutdown(task.getId(), reasonFormat, args);
     }
     catch (Throwable e) {
-      // If task runner shutdown fails, continue with the task shutdown routine. We'll come back and try to
-      // shut it down again later in manageInternalPostCritical, once it's removed from the "tasks" map.
+      // If task runner shutdown fails, continue with the task shutdown routine.
       log.warn(e, "TaskRunner failed to cleanup task after completion: %s", task.getId());
     }
 
     removeTaskLock(task);
     requestManagement();
+
+    log.info("Completed notifyStatus for task[%s] with status[%s]", task.getId(), taskStatus.getStatusCode());
```

Comment:
I think it is fine to log the full status with the error message, etc.

```suggestion
    log.info("Completed notifyStatus for task[%s] with status[%s].", task.getId(), taskStatus);
```